### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by gianni-rosato** [gianni-rosato/svt-av1-psy#48] [Announcements]
Hi again! We're just barely trailing mainline's recent release to announce some exciting new changes in **SVT-AV1-PSY v2.1.0**! 🎉

### Documentation
- Additional PNG images have been transcoded to WebP
- [Variance boost docs](https://github.com/gianni-rosato/svt-av1-psy/blob/master/Docs/Appendix-Variance-Boost.md) feature AVIF & WebP images
- Added Staxrip to featured projects in README. Welcome aboard!

### Bug Fixes
- Removed ANSI escape color sequences for Windows builds (#46)

### Other
- This major release doesn't have a micro-release letter, & is just labelled `PSY Release: N/A` - if we're doing a release in line with mainline, PSY release lettering will start at N/A & progress to A, B, etc from there.

### API updates
- One config parameter added within the padding size. Config param structure size remains unchanged
- Presets 6 and 12 are now pointing to presets 7 and 13 respectively due to the lack of spacing between the presets
- Further preset shuffling is being discussed in #2152

### Encoder
- Added variance boost support to improve visual quality for the tune vq mode
- Improve the tradeoffs for the random access mode across presets:
- Speedup of 12-40% presets M0, M3, M5 and M6 while maintaining similar quality levels
- Improved the compression efficiency of presets M11-M13 by 1-2%
- Added ARM optimizations for functions with c-only equivalent

### Cleanup, build, bug fixes & documentation
- Use nasm as a default assembler and yasm as a fallback
- Fix performance regression for systems with multiple processor groups
- Enable building SvtAv1ApiTests and SvtAv1E2ETests for ARM
- Added variance boost documentation
- Added a mailmap file to map duplicate git generated emails to the appropriate author

*Thanks for using SVT-AV1-PSY!* ♥️

**Full Changelog**: https://github.com/gianni-rosato/svt-av1-psy/commits/v2.1.0

*Compiled binaries coming soon*

<hr /><em>This discussion was created from the release <a href='https://github.com/gianni-rosato/svt-av1-psy/releases/tag/v2.1.0'>v2.1.0</a>.</em>

**Comment by shodan5000** [comment]
When can a Windows build be expected?

**Reply by gitoss** [reply]
Staxrip has the 2.1.0 exe: https://github.com/staxrip/staxrip/releases/tag/v2.40.0

Stil, having an immediate Windows psy binary in the original repo would be very welcome for upcoming releases.
